presto-main/src/main/java/com/facebook/presto/execution/scheduler/NodeSchedulerConfig.java

```diff
@@ -20,15 +20,18 @@
 import javax.validation.constraints.Min;
 import javax.validation.constraints.NotNull;
 
-@DefunctConfig("node-scheduler.location-aware-scheduling-enabled")
+@DefunctConfig({
+        "node-scheduler.location-aware-scheduling-enabled",
+        "node-scheduler.max-pending-splits-per-node-per-task"
```

Comment:
Maybe make this a `@LegacyConfig` instead? Otherwise the server will fail to start if you have the old setting